crates/aptos-telemetry-service/src/lib.rs

```diff
@@ -77,15 +79,13 @@ impl AptosTelemetryServiceArgs {
         );
         let validators_cache = PeerSetCache::new(aptos_infallible::RwLock::new(HashMap::new()));
         let vfns_cache = PeerSetCache::new(aptos_infallible::RwLock::new(HashMap::new()));
-        let pfns_cache = Arc::new(aptos_infallible::RwLock::new(HashMap::new()));
-
-        pfns_cache.write().clone_from(&config.pfn_allowlist);
 
         let context = Context::new(
             &config,
             validators_cache.clone(),
             vfns_cache.clone(),
-            pfns_cache.clone(),
+            config.pfn_allowlist.clone(),
+            config.log_env_map.clone(),
```

Comment:
Just so that I understand correctly, this is a map of peer id to the corresponding log level and will allow us to override the log level per peer?

Reply:
yes, we should configure log level per peer per chain.

Reply:
Like below:
```
log_env_map:
  4:
    "86ebcca4d68b381e5d99d5ae448212c0122de627efbc580a7a2e8cd364b94430": "debug"
```